### PR TITLE
Feat stim

### PR DESCRIPTION
**Pull request by gburnett-stanford**

### CODE REVIEW

File: stim.proto [hunk 1]
```diff
@@ -0,0 +1,571 @@
+syntax = "proto3";
+package openmind;
+
+option csharp_namespace = "OpenMind";
+
+import "summit.proto";
+
+service StimService {
```
Comment (jeffherron) [inline stim.proto:8]: Weren't we going to put the stim commands in the 'device' service as opposed to creating a new one?

**Review by jeffherron — COMMENTED** [review]
I think we were going to integrate these into the device service instead of creating a separate service right? Otherwise I think this all looks great!

File: stim.proto (hunk 1 above)
Comment (broarr) [reply stim.proto:8]: That's my recollection too!

**Review by broarr — DISMISSED** [review]
This looks great! We're trying to follow a [resource oriented design](https://cloud.google.com/apis/design/resources#resources). You can think of resources as the nouns of a system. Stim is a verb, the noun for our resource is the Summit RC+S (or in OMNI parlance, a device). That's why we want to add the stim endpoints into the device service. Does that make sense?

File: stim.proto (hunk 1 above)
Comment (gburnett-stanford) [reply stim.proto]: Sorry for the delay. I just pushed changes for this. I deleted stim.proto and moved everything into device.proto. Now I have a minor design question: device.proto now has two services (DeviceManagerService and StimService). Previously all the message types were loosely written in order form highest level to lowest level. But the crude way I pasted in StimService breaks that (although within StimService, that order is preserved). Is that okay? Should I move the services to the top and re-order the message types? Does it matter? 

If that didn't make sense, we can talk about it in the meeting tomorrow.

**Comment by gburnett-stanford** [comment]
> This looks great! We're trying to follow a [resource oriented design](https://cloud.google.com/apis/design/resources#resources). You can think of resources as the nouns of a system. Stim is a verb, the noun for our resource is the Summit RC+S (or in OMNI parlance, a device). That's why we want to add the stim endpoints into the device service. Does that make sense?

That totally makes sense!

**Review by jeffherron — CHANGES_REQUESTED** [review]
Hey Gary, we're in the right file but I don't think we want the StimService to be separate from the device service - i.e. we want all the RPCs to be calls that can be made through the device service. I think the messages are all good, so really all we have to do is just move the stimservice rpcs into the device service rpc list. Let me know if you weant to chat about this at all!

**Comment by gburnett-stanford** [comment]
okay third time is the charm. i just removed the StimService and moved all those commands into DeviceManagerService.

**Review by jeffherron — APPROVED** [review]

**Pull request merged**